Solar.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.homepage     = "http://github.com/ceek/solar"
   s.license      = { :type => "MIT", :file => "LICENSE" }
   s.author             = { "Chris Howell" => "chris.kevin.howell@gmail.com" }
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '9.0'
```

Comment:
As much as we need to increase the minimum deployment target to stay up-to-date with Xcode requirements, it is technically a backwards-incompatible change, requiring a major version bump. A major version bump seems reasonable to me at this point.

@ceeK, it would be helpful to get this update to CocoaPods trunk as well once it lands.